playwright/src/fixtures/content.page.ts

```diff
@@ -143,9 +146,21 @@ class ContentPage {
 
   async scrolltotop() {
     // Scroll to top of content area and click
-    const body = await this.body.boundingBox()
-    await this.page.mouse.wheel(body.x + 100, body.y + 100)
-    await this.page.mouse.click(body.x + 100, body.y + 100)
+
+    const x1 = this.page.context().browser().browserType
+    console.log(x1)
+    
+    // console.log(test.info.name)
+    
```

Comment:
@gregfitch I want to get the browser name to do some browser specific actions here. Could you please let me know the right way to call for a browser name?

Reply:
the easiest way is to use the `browserName` fixture ( https://playwright.dev/docs/api/class-fixtures#fixtures-browser-name )

Reply:
@gregfitch do you mean like this in the test? https://github.com/openstax/rex-web/blob/dc5d473cea7bcae9c9449cc629f002d178d1a473/playwright/tests/rex-test/rex.behaviorspec.ts#L20

I am looking for something that can be called from the content.page.ts. Firefox is behaving slightly different after highlighting text, so I would want to do things diffently for firefox alone after highlighting text. Are there other options or can we somehow call that fixture inside the content.page.ts

Reply:
In that case you may want to use either the javascript user agent property to get the user agent string (`window.navigator.userAgent` contains 'Gecko' or 'Firefox') or the browser type attached to the Browser Playwright object (https://playwright.dev/docs/api/class-browser#browser-browser-type is 'firefox').